### ISSUE TITLE
Two mouses / Trackball support

### ISSUE THREAD

**Issue by bodobear** [MiSTer-devel/Genesis_MiSTer#165]
Hello,
would it be possible to add support for two mouses /trackballs ?
Marble Madness (jap) is playable with two mouses on the genesis.
That would be a great feature :)

Bodo

**Comment by iequalshane** [comment]
If you have SNAC and a Sega Mouse you can do this using the Sega Mouse and a USB mouse. The framework doesn't support 2 USB mice that I'm aware of.

**Comment by sorgelig** [comment]
Correct. Framework supports only aggregated mouse, just like all modern computer systems. There is no plans to change it.

**Comment by bodobear** [comment]
Ok thanks for the info !
